lib/puppet/defaults.rb

```diff
@@ -1085,6 +1085,16 @@ def self.default_vendormoduledir
       :desc       => "The default TTL for new certificates.
       #{AS_DURATION}",
     },
+    :crl_refresh_interval => {
+      :type       => :duration,
+      :desc       => "How often the Puppet agent refreshes its local CRL. By default,
```

Comment:
Not sure how beneficial it would be to point out that this isn't independent of the agent run interval? For example, setting this to 1 hour if you have a run interval of 12 hours means that you're still only going to refresh every 12 hours. Setting this to 1 hour with the default 30 minute run interval would mean that every other agent run would attempt to refresh the CRL. Unless I missed something?

Reply:
Good point, let me update the description